lesson_02_homework.rb

```diff
@@ -26,21 +29,35 @@ def full_name(f, l)
   puts f + l
 end
 
-puts full_nam("bubba", "chuck")
+puts full_name("bubba", "chuck")
 
 
 # Coding
 
 
 ## Enter your code for your 'add' method below
+def add(num1, num2)
+  puts "Total numer is #{add = num1 + num2}"
```

Comment:
🔨  @Alisher778 using `puts` in the method body makes the method less flexible - if you remove the `puts` you will achieve the same result without forcing the method to print information out to the console.

Also this method should only return the sum of the two numbers, and not any strings. If you want to create a message such as the "Total number is ...." then you should do that outside of the method